docker-app/worker_wrapper/wrapper.py

```diff
@@ -262,10 +262,10 @@ def _run_docker(
             detach=True,
             mem_limit=config.WORKER_QGIS_MEMORY_LIMIT,
             cpu_shares=config.WORKER_QGIS_CPU_SHARES,
-            labels={
-                "worker": self.job_class.type,
-                "project_id": str(self.job.project_id),
-            }
+            labels=[
+                "worker",
+                str(self.job.project_id),
+            ]
```

Comment:
Why would you prefer lists instead of dictionaries? I think it's easier to consume the labels if they are named.

They are both supported https://docker-py.readthedocs.io/en/stable/containers.html .

Reply:
Yes agree. I started with a dictionary: "worker: job.type". Then turned out that filtering only after the key did not work. I.e. `containers.list(filters={label: "worker}). Because I had a dynamic value I could not use the value in filtering. But I could just use {"type": "worker", "project_id": "..."} then filter after "type=worker" . 

BTW I removed the project_id in the recent revision. Should I let it there?

Reply:
I think it is nice to have as much meta data as possible, so if you can resurrect `project_id`.

> Because I had a dynamic value I could not use the value in filtering. But I could just use {"type": "worker", "project_id": "..."} then filter after "type=worker" .

According to the docs it should work: https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.ContainerCollection.list

Diid you try with a more recent `docker-py` version if there is a bugfix?

Reply:
No, willl try